src/lib/HST/Frontend/Build.hs

```diff
@@ -0,0 +1,33 @@
+module HST.Frontend.Build where
```

Comment:
I would add comments to all modules because otherwise it is very hard to understand the code.